blue-note/assets/images/thin-stripes.svg

```diff
@@ -0,0 +1,31 @@
+<svg width="1322" height="1044" viewBox="0 0 1322 1044" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
I would remove the `width` and `height` parameters since the size should be defined by the image block and the `viewBox` _should_ be enough to position things relatively in the `clipPath` definition.

I think `fill` can also be removed since we're just providing definitions.

And `xlink` can probably be removed too since we aren't using any of its features.